py/picca/delta_extraction/data_catalogues/desi_healpix.py

```diff
@@ -4,25 +4,16 @@
 import os
 import multiprocessing
 
-
 import fitsio
-import healpy
 import numpy as np
 
-from picca.delta_extraction.astronomical_objects.desi_forest import DesiForest
-from picca.delta_extraction.astronomical_objects.desi_pk1d_forest import DesiPk1dForest
-from picca.delta_extraction.astronomical_objects.forest import Forest
-from picca.delta_extraction.data_catalogues.desi_data import DesiData, defaults, accepted_options
+from picca.delta_extraction.data_catalogues.desi_data import DesiData
+from picca.delta_extraction.data_catalogues.desi_data import (# pylint: disable=unused-import
```

Comment:
the split is just for line length? And the pragma is for pylint not to complain there because the import is multi-line?

Reply:
The split is because one is used in the module and the others are simply loaded (they are necessary imports but are not used inside the module). The pragma is for pylint to not complain about the fact the two imported variables are not used inside the module

Reply:
that's because they are only used for child classes? Or why would we elsewise import them here if they aren't used?

Reply:
They are (or could be) used by child classes. In any case, they are also used by `Config` in order to enforce correct parameters in the config file and give early errors